### ISSUE TITLE
Crash when opening a file and trying to delete it completely with backspace key

### ISSUE THREAD

**Issue by Ribodou** [curlpipe/ox#69]
**Describe the bug**
Ox may crash when the user is pressing the "backspace" key : if the file is empty when pressing the backspace key, an "index out of bounds" error occurs. 

**To Reproduce**
open ox : `ox`
=> press backspace two times
=> crash

This also happens with an existing file, eg. when calling `ox txt.txt`
**Expected behavior**
ox shouldn't crash. In fact, nothing should happen.

**Desktop:**
 - OS: Debian 10
- I installed ox with cargo

Please find the stacktrace below :
```
thread 'main' panicked at 'index out of bounds: the len is 0 but the index is 0', /usr/src/rustc-1.41.1/src/libcore/slice/mod.rs:2806:10
stack backtrace:
   0:     0x5633b4b5e667 - backtrace::backtrace::libunwind::trace::h8cc7bd6ef9eecc96
                               at /usr/src/rustc-1.41.1/vendor/backtrace/src/backtrace/libunwind.rs:88
   1:     0x5633b4b5e667 - backtrace::backtrace::trace_unsynchronized::h6d4579bff574314f
                               at /usr/src/rustc-1.41.1/vendor/backtrace/src/backtrace/mod.rs:66
   2:     0x5633b4b5e667 - std::sys_common::backtrace::_print_fmt::h90c5eb8ce5c21ab3
                               at src/libstd/sys_common/backtrace.rs:84
   3:     0x5633b4b5e667 - <std::sys_common::backtrace::_print::DisplayBacktrace as core::fmt::Display>::fmt::hd925cae363a8eae6
                               at src/libstd/sys_common/backtrace.rs:61
   4:     0x5633b4ad758d - core::fmt::write::h7f27f0f32a72efde
                               at src/libcore/fmt/mod.rs:1025
   5:     0x5633b4b5e0d4 - std::io::Write::write_fmt::h248d5ddff30f2014
                               at src/libstd/io/mod.rs:1426
   6:     0x5633b4b6060b - std::sys_common::backtrace::_print::h998846397d49de77
                               at src/libstd/sys_common/backtrace.rs:65
   7:     0x5633b4b6060b - std::sys_common::backtrace::print::he55a1a23a73d03c9
                               at src/libstd/sys_common/backtrace.rs:50
   8:     0x5633b4b6060b - std::panicking::default_hook::{{closure}}::h80dc9a42e1547e54
                               at src/libstd/panicking.rs:193
   9:     0x5633b4b601aa - std::panicking::default_hook::h50769e30da5feef3
                               at src/libstd/panicking.rs:210
  10:     0x5633b4b601aa - std::panicking::rust_panic_with_hook::hd0b3a149883f4cfb
                               at src/libstd/panicking.rs:471
  11:     0x5633b4b607ad - rust_begin_unwind
                               at src/libstd/panicking.rs:375
  12:     0x5633b4adb37d - core::panicking::panic_fmt::h2c5b80195dbe1b07
                               at src/libcore/panicking.rs:84
  13:     0x5633b4adb344 - core::panicking::panic_bounds_check::h947025a66656df4b
                               at src/libcore/panicking.rs:62
  14:     0x5633b4a5a9b8 - ox::editor::Editor::execute::hd2ebd2d2c264076c
  15:     0x5633b4a6e8a4 - ox::main::hf9d8af8cefc5f1fa
  16:     0x5633b4a2a253 - std::rt::lang_start::{{closure}}::h30625ffaf4283e4e
  17:     0x5633b4a777ab - main
  18:     0x7f7f87b6c09b - __libc_start_main
  19:     0x5633b4a2718a - _start
  20:                0x0 - <unknown>
```

**Comment by Asetyan** [comment]
Also happens here.

Desktop :
- OS: Ubuntu 20.04.1
- I installed ox with cargo with this command : cargo install --git https://github.com/curlpipe/ox

**Comment by curlpipe** [comment]
Thanks for the report. This is a duplicate, I've already fixed this in 0.2.7 which will be released in the next week.
